### PR TITLE
2021 05 08 deeamon128 javascript practice

### PR DESCRIPTION
**Pull request by deeamon128**
weather app with API

### CODE REVIEW

**Review by Nicolleta1981 — APPROVED** [review]
skycons.js is impressive over 700k lines and so much math..is impressive work that is for sure. Style around 200k lines no grid only flex..if works than yes you rock

**Comment by deeamon128** [comment]
> skycons.js is impressive over 700k lines and so much math..is impressive work that is for sure. Style around 200k lines no grid only flex..if works than yes you rock

wish that file would be mine, but is just a code with icons I found online. :D Many thanks for your review

File: web/deeamon128/javascript/weather-app/style.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  height: 100vh;
+  background: linear-gradient(rgb(26,39,96), rgb(139,93,158));
```
Comment (biggicode) [inline web/deeamon128/javascript/weather-app/style.css:9]: some spaces for readability

**Review by biggicode — APPROVED** [review]

**Review by Adrian01C — COMMENTED** [review]
I can't approve this since I don't know java  but looks  cool.

File: web/deeamon128/javascript/weather-app/app.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+window.addEventListener("load", () => {
+  let locationTimeZone = document.querySelector('.location-timezone');
+  let temperatureDegree = document.querySelector('.temperature-degree');
+  let temperatureDescription = document.querySelector('.temperature-description');
+
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(position => {
+
+      const api = `https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/London?unitGroup=uk&key=E9R3A3G5GD6CPRK8S7YZBLCVB`;
```
Comment (QuintyHH) [inline web/deeamon128/javascript/weather-app/app.js:9]: Normally, you would want to extract the URL params into variables.
```
const country = 'uk'
const APIkey = 'E9R3A3G5GD6CPRK8S7YZBLCVB'

const api = `https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/London?unitGroup=${country}&key=${APIkey}`
```

**Review by QuintyHH — APPROVED** [review]
Very nice. Left a single comment, for future references.

**Review by simonamarcu2 — APPROVED** [review]
Great job!

File: web/deeamon128/javascript/weather-app/app.js (hunk 2 above)
Comment (deeamon128) [reply web/deeamon128/javascript/weather-app/app.js:9]: thank you

**Pull request merged**